core/src/main/java/org/apache/carbondata/core/util/CarbonTimeStatisticsFactory.java

```diff
@@ -17,13 +17,14 @@
 
 package org.apache.carbondata.core.util;
 
-import org.apache.carbondata.core.carbon.querystatistics.DriverQueryStatisticsRecorder;
+import org.apache.carbondata.core.carbon.querystatistics.*;
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
 
 public class CarbonTimeStatisticsFactory {
```

Comment:
This class is the factory class for `QueryStatisticsRecorder`, so rename it to `QueryStatisticsRecorderFactory`.